appveyor.yml

```diff
@@ -16,19 +16,6 @@ environment:
     # https://www.appveyor.com/docs/windows-images-software/#python
 
     - PYTHON: "C:\\Python37"
-      TARGET: "test"

```

Comment:
Hey James, I'm still unfamiliar with the appveyor stuff. Could you just mention why these lines are being pulled? Thanks!

Reply:
Yeah, so the main idea behind pulling these lines is that this PR moves most of the test jobs to github actions.  The main reason for the move is that while AppVeyor offers free builds for open-source projects, they only allow one job to run at a time.  As a result, the full InVEST test suite within AppVeyor takes almost an hour, and if you have any other build jobs running at all they start queueing.

By moving the tests to github actions, we're able to run our _entire_ test suite (including GUI tests!) in parallel, all while allowing the appveyor-based builds to happen independently.

Reply:
Ahhh, I see, I see. Thanks!